modules/formic/assets/ships/formicSmall/formicSmall.json

```diff
@@ -0,0 +1,1164 @@
+            "x": 0.23846153914928436,
+            "y": 0.23846152424812317
+          },
+          {
+            "x": 0.17307692766189575,
+            "y": 0.23461538553237915
+          },
+          {
+            "x": 0.13076923787593842,
+            "y": 0.2192307710647583
+          },
+          {
+            "x": 0.13076923787593842,
+            "y": 0.23846152424812317
+          },
+          {
+            "x": 0.003846153849735856,
+            "y": 0.1884615421295166
+          },
+          {
+            "x": 0.0615384615957737,
+            "y": 0.22692307829856873
+          },
+          {
+            "x": 0.057692307978868484,
+            "y": 0.24230769276618958
+          },
+          {
+            "x": 0.03076923079788685,
+            "y": 0.24230769276618958
+          },
+          {
+            "x": 0.057692307978868484,
+            "y": 0.26923075318336487
+          },
+          {
+            "x": 0.05000000074505806,
+            "y": 0.2846153676509857
+          },
+          {
+            "x": 0.01923076994717121,
+            "y": 0.2846153676509857
+          },
+          {
+            "x": 0.01923076994717121,
+            "y": 0.36153843998908997
+          },
+          {
+            "x": 0.05384615436196327,
+            "y": 0.3730769157409668
+          },
+          {
+            "x": 0.05384615436196327,
+            "y": 0.39230766892433167
+          },
+          {
+            "x": 0.023076923564076424,
+            "y": 0.4076923131942749
+          },
+          {
+            "x": 0.06538461893796921,
+            "y": 0.41538459062576294
+          },
+          {
+            "x": 0,
+            "y": 0.4692307710647583
+          },
+          {
+            "x": 0.11923076957464218,
+            "y": 0.41923075914382935
+          },
+          {
+            "x": 0.13076923787593842,
+            "y": 0.4423076808452606
+          },
+          {
+            "x": 0.18076923489570618,
+            "y": 0.41923075914382935
+          },
+          {
+            "x": 0.3192307651042938,
+            "y": 0.4307692050933838
+          },
+          {
+            "x": 0.33076924085617065,
+            "y": 0.47692304849624634
+          },
+          {
+            "x": 0.36153846979141235,
+            "y": 0.5115384459495544
+          },
+          {
+            "x": 0.3192307651042938,
+            "y": 0.5269230604171753
+          },
+          {
+            "x": 0.3153846263885498,
+            "y": 0.550000011920929
+          },
+          {
+            "x": 0.3692307770252228,
+            "y": 0.5769230723381042
+          },
+          {
+            "x": 0.3692307770252228,
+            "y": 0.5923076868057251
+          },
+          {
+            "x": 0.2807692289352417,
+            "y": 0.6192307472229004
+          },
+          {
+            "x": 0.2730769217014313,
+            "y": 0.6346153616905212
+          }
+        ]
+      }
+    ]
+  }
+}
```

Comment:
Missing newline at the end of the file (bulk reformat might handle this but keep a tiny eye out for this and other good practices)